Tasks/AndroidSigningV2/package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "@types/node": "^6.0.0",
-    "securefiles-common": "file:../../_build/Tasks/Common/securefiles-common-1.0.0.tgz",
-    "vsts-task-lib": "2.4.0"
+    "azure-pipelines-task-lib": "^2.8.0",
+    "securefiles-common": "file:../../_build/Tasks/Common/securefiles-common-1.0.0.tgz"
```

Comment:
It's a shame our current design requires updating and testing older versions of tasks. Maybe we should make securefiles and ios-signing npm packages.

Reply:
Yes, 100% agree.  The common pattern is problematic.  Better as npm with semver

Reply:
Yes either we need to make them npm packages or if they don't hold much logic, keep them with the task itself. The common model doesn't really work with the number of tasks and versions we have.